compiler/src/dmd/denum.d

```diff
@@ -66,6 +63,8 @@ extern (C++) final class EnumDeclaration : ScopeDsymbol
     import dmd.common.bitfields : generateBitFields;
     mixin(generateBitFields!(BitFields, ubyte));
 
+    Symbol* sinit;
+
```

Comment:
unrelated change

Reply:
Just grouped it with the other field declarations. The layout hasn't changed.

Reply:
frontend.h needed to be updated for this change. This PR was merged with failing CI.